### PR TITLE
Draggable map fix. Click to go and cancel button for autopilot.

### PR DESCRIPTION
**Pull request by epoponin**
- Draggable button now works
- Click to point autopilot into specific location (in locked map mode)
- Added cancel button to autopilot popup dialog

### CODE REVIEW

**Comment by h4v1nfun** [comment]
LGTM

@iam4x

**Comment by 93732** [comment]
@epoponin 

how do i compile/build your version?

**Comment by epoponin** [comment]
Anyone who don't want to wait until merge I have built release on my fork
https://github.com/epoponin/pokemongo-webspoof/releases/download/1.2.0/pokemongo-webspoof-1.2.app.tar.gz

**Comment by AndersBrohus** [comment]
So @epoponin does this also fix the issue where we get teleported back to our real locatiob?

**Comment by epoponin** [comment]
@AndersBrohus im afraid no. It just adds few new features and map fix.

**Comment by clburlison** [comment]
@epoponin I've been playing with this PR for the last 24 hours and it has been great. One recommendation, it might make more sense if it was a right click to to enable the autopilot popup. Reason being: on trackpads Apple users are trained that left click is for dragging/panning. However, It doesn't take long to adapt to the current implementation.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 ![Example](./example.gif)
 
-* :arrow_right: Download last release [pokemongo-webspoof.app.tar.gz (v1.1.1)](https://github.com/iam4x/pokemongo-webspoof/releases/download/v1.1.1/pokemongo-webspoof-v111.app.tar.gz)
-* :arrow_right: Changelog is available [here](https://github.com/iam4x/pokemongo-webspoof/releases)
+* :arrow_right: Download last release [pokemongo-webspoof.app.tar.gz (v1.2)](https://github.com/epoponin/pokemongo-webspoof/releases/download/1.2.0/pokemongo-webspoof-1.2.app.tar.gz)
+* :arrow_right: Changelog is available [here](https://github.com/epoponin/pokemongo-webspoof/releases)
```
Comment (clburlison) [inline README.md]: You should revert this commit. So iam4x doesn't have to cherry pick your PR.

**Comment by epoponin** [comment]
@clburlison reverted last commit (readme)

**Comment by iam4x** [comment]
Thank's a lot :+1:

**Pull request merged**